code/controllers/subsystem/ticker.dm

```diff
@@ -489,3 +473,45 @@ var/datum/subsystem/ticker/ticker
 		return
 	spawn(-1) //compiling a map can lock up the mc for 30 to 60 seconds if we don't spawn
 		maprotate()
+
+/datum/subsystem/ticker/proc/announce_statistics() //Display five random round-end stats, plus a few guaranteed ones.
+	world << "<br>"
+	world << "<span class='big'><b>Round Statistics Report</b></span>"
+
+	//Get info for the generic stats
+	if(SSshuttle.emergency.mode >= SHUTTLE_ESCAPE)
+		station_evacuated = 1
+	var/datum/station_state/end_state = new /datum/station_state()
+	end_state.count()
+	var/station_integrity = min(round( 100 * start_state.score(end_state), 0.1), 100)
+
+	//Generic stats: time, integrity, population...
+	world << "<br><b>Shift Duration:</b> [round(world.time / 36000)]:[add_zero("[world.time / 600 % 60]", 2)]:[world.time / 100 % 6][world.time / 100 % 10]"
+	world << "<br><b>Station Integrity:</b> [mode.station_was_nuked ? "<font color='red'>Destroyed</font>" : "[station_integrity]%"]"
+	world << "<br><b>Total Population:</b> [joined_player_list.len]"
+	world << "<br><b>Survival Rate:</b> [num_survivors] ([round((num_survivors/joined_player_list.len)*100, 0.1)]%)"
+	if(station_evacuated)
+		world << "<br><b>Evacuation Rate:</b> [num_escapees] ([round((num_escapees/joined_player_list.len)*100, 0.1)]%"
+
+	/* Uncomment this block to enable announcement of the people with the highest and lowest kill count
```

Comment:
make this a a config setting, don't leave it commented out

downstreams may want it.